srml/timestamp/src/lib.rs

```diff
@@ -227,6 +227,15 @@ decl_module! {
 			<T::OnTimestampSet as OnTimestampSet<_>>::on_timestamp_set(now);
 		}
 
+		// Manage upgrade. Remove after all networks upgraded.
```

Comment:
Can you link to https://github.com/paritytech/substrate/issues/2133 in the code? (our policy that TODOs all have issues, and this is basically a TODO)